### DISCUSSION TITLE
I can't create an account

### DISCUSSION THREAD

**Discussion by ZedroPilves** [vercel/community#3759] [Account Issue] [answered]
### Question

My web development teacher asked our class to create an account in vercel linked with our githubs but i can't  all the accounts are blocked, i tried with my secondary email, it got created but when i try to login with it a error message appears and i didn't even created a password

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble creating a new account. Due to the sensitive nature of account infomation, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will open a new case with the support team.

**Comment by Aierbote** [comment]
I my case  I got the `There is no Vercel account linked with this GitHub account.` during the *Login With Github* when I had to create an account.

I gave the authorisations at the first login, but it seems a combination of:
1. deactivate my adblocker plugin
2. going into github settings to remove authorisation (to reset the first login authorisation later on)
3. going to the vercel homepage to be sure to deactivate adblocker from the "root" in all the website

I can't remember right now if I repeated step 2 a second time after step 3, but anyway the lastest try to Login With Github as the new first login worked in the end. Hope this could be of any help for others.
